src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 # =========================================================================
 # Spring-related.
 spring.profiles.active=default
-spring.config.import=classpath:custom.properties
+spring.config.import=classpath:custom.properties, optional:file:${user.home}/.${user.name}-conf/uh-groupings-ui-overrides.properties
```

Comment:
This file does not need the overrides file either.

Reply:
Done